src/ExceptionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Google\Authenticator;
+
+/**
+ * Indicates the interface is coming from this package.
+ *
+ * @author Iltar van der Berg <kjarli@gmail.com>
+ */
+interface ExceptionInterface
```

Comment:
Oh, I forgot to catch the exceptions based on the new request class, I still have to add those! 

This interface was supposed to be for that purpose, but now that I think of it, maybe not really needed